src/pageSizeSelector.js

```diff
@@ -47,15 +47,15 @@ var defaults = {
  * @param {Object} [config.items] - available options
  * @returns {pageSizeSelector}
  */
-var pageSizeSelectorFactory = function pageSizeSelectorFactory(config) {
-    var pageSizeSelectorSpecs = {
-        setSelectedOption: function setSelectedOption() {
-            var options = this.config.options;
-            var defaultSize = this.config.defaultSize;
+export default function pageSizeSelectorFactory(config) {
+    const pageSizeSelectorSpecs = {
+        setSelectedOption() {
+            const options = this.config.options;
+            const defaultSize = parseInt(this.config.defaultSize, 10);
```

Comment:
the config was also affected? :(

Reply:
Well, not necessarily. But it is better to safeguard it.